recipe/meta.yaml

```diff
@@ -37,12 +35,32 @@ requirements:
     - pyprind >=2.9.3
     - py_stringmatching >=0.2.1
 
+{% set tests_to_ignore = "\"" %}
+# test_valid_join is called three times and results in, inter alia:
+# E       fixture 'scenario' not found
+# >       available fixtures: cache, capfd, capfdbinary, caplog,
+#           capsys, capsysbinary, doctest_namespace, monkeypatch,
+#           pytestconfig, record_property, record_testsuite_property,
+#           record_xml_attribute, recwarn, tmp_path, tmp_path_factory,
+#           tmpdir, tmpdir_factory
+# >       use 'pytest --fixtures [testpath]' for help on them.
```

Comment:
```suggestion
# skip test_valid_join (in 3 test files), marked @unittest.skip("Not a test")
```

Reply:
See: https://github.com/search?q=repo%3Aanhaidgroup%2Fpy_stringsimjoin+test_valid_join&type=code

Reply:
Yep, that seems fine.  I thought the actual error might be useful to someone. 😄

Reply:
The function is not a test, and if we run it with `pytest`, the tool searches for each of the parameters a fixture named like the parameter (`scenario`, `tok`, etc), and it does not find it, because it does not exist, not because it is an error. So it's more like we tell `pytest` "don't run this because it's not a test".